Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -93,6 +95,13 @@ - (id)initWithAnalytics:(SEGAnalytics *)analytics
         self.serialQueue = seg_dispatch_queue_create_specific("io.segment.analytics.segmentio", DISPATCH_QUEUE_SERIAL);
         self.flushTaskID = UIBackgroundTaskInvalid;
         self.analytics = analytics;
+        // Check for previous queue/track data in NSUserDefaults and store to disk
```

Comment:
Would dispatching to a background queue look like wrapping lines 99 ->104 or would that be inside the body of the transferQueue and transferTraits methods? @tonyxiao